pkg/configflags/helpers.go

```diff
@@ -0,0 +1,43 @@
+package configflags
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// ArgsWithPrefix filters arguments by prefix and collects values.
+func ArgsWithPrefix(args map[string][]string, prefix string) map[string][]string {
+	filtered := map[string][]string{}
+	for key, slice := range args {
+		if !strings.HasPrefix(key, "audit-") {
```

Comment:
This still isn't addressed, you want to filter by `prefix` not `audit-`

Reply:
we want to filter by `audit-`, that's what the old code tried to do

Reply:
oh, now I get it :D

Reply:
fixed